rb/spec/integration/selenium/webdriver/bidi/log_inspector_spec.rb

```diff
@@ -21,13 +21,13 @@
 
 module Selenium
   module WebDriver
-    class BiDi
-      describe LogInspector, exclusive: {browser: %i[chrome firefox]} do
+    class BiDi # rubocop:disable Metrics/ClassLength
+      describe LogInspector, only: {browser: %i[chrome edge firefox]} do
         before do
           @page = '/bidi/logEntryAdded.html'
```

Comment:
for this, we should prefer `let` instead of setting an instance variable in a before hook: `let(:page) { '/bidi/logEntryAdded.html' }`